docs/apps/components/sub-form+.md

```diff
@@ -0,0 +1,138 @@
+|`this.getComponent`|Returns a component whose id has been passed as a parameter|`(component_ID: String) : Object`|
+|`this.hide`|Hides the field|`(fields: String`&#124;`String[])`|
+|`this.show`|Displays the field|`(fields: String \| String[])`|
+|`this.disable`| Disable sub-form+ field from user interaction|`(fields: String`&#124;`String[])`|
+|`this.enable`| Enable sub-form+ field from user interaction|`(fields: String`&#124;`String[])`|
+
+!!! Info
+    1. The show() and hide() methods can also be used to control the visibility of a sub-form+ component in response to user input.
+    2. **Fields** refers to a **component ID** or a **list of component IDs**. You can fetch the ID from the **Component Attribute** panel in the Page Builder.
+    3. Before using **this.show(fields)**, make sure the component is **hidden**. This can be done using **this.hide(fields)** or by **enabling** the **Hidden checkbox** in the Component Attribute panel.
+    4. Before using **this.enable(fields)**, make sure the component is **disabled**. This can be done using **this.disable(fields)** or by **enabling** the **Disabled checkbox** in the Component Attribute panel.
+
+#### Steps to use the methods for the Page Builder components
+
+1. Go to Form Attribute :material-menu-right: Action Panel :material-menu-right: Setting :material-menu-right: (Mounted | refresh | click 'Add action').
+2. Write the method/code as shown in the **Example** below.
+3. Click on `Save`.
+4. On the main screen click on `Save` again.
+5. Click on `Preview` to see the code in action.
+<img src= "/apps/components/img/subformplus2.png">
+
+!!! Example
+    1. `this.addClassName(fields, className)`
+          * Go to Form Attribute :material-menu-right: Style Sheets :material-menu-right: add the class
+            ```
+            .abc{ // abc is the class name
+            background-color: red;
+            }
+            ```
+          * Follow the steps mentioned above, under Steps to use the methods for the Page Builder components
+          * ```
+            this.addClassName('subform_e4cjz3nf', 'abc')
+            ```
+
+    2. `this.removeClassName(fields, className)`
+        ```js
+        this.removeClassName('subform_e4cjz3nf', 'abc')
+        ```
+
+    3. `this.hide(fields)`
+        ```js
+        var fields= ['subform_e4cjz3nf']
+        this.hide(fields)
+        ```
+    
+    4. `this.show(fields)`
+        ```js
+        var fields= ['subform_e4cjz3nf']
+        this.show(fields)
+        ```
+    
+    5. `this.disable(['fields'])`
+        ```js
+        this.disable(['subform_e4cjz3nf'])
+        ```
+    
+    6. `this.enable(['fields'])`
+        ```js
+        this.enable(['subform_e4cjz3nf'])
+        ```
+
+    7.  `this.getComponent('component_ID')`
+        ```js
+        var subname = this.getComponent('subform_e4cjz3nf');
+        console.log('getComponent', subname);
+        ```
+
+## Config
```

Comment:
Add name field
Add Label Wrap field
Remove Virtual Table field

Under Attribute Action
======================
Remove show word count
Change image to correctly represent the attribute actions of Sub Form +

Reply:
done